soc/mediatek/mtk_adsp/soc.yml

```diff
@@ -1,6 +1,21 @@
 family:
   - name: mtk_adsp
```

Comment:
with all those changes, is the family still correct? mt8195 is the container of adsp 'cpucluster', so the family can't be named mtk_adsp, no?

Reply:
Indeed, arguably no, for the same reason as the board quibble above.  But again, this isn't[1] intended to be a framework for all Zephyr firmware to run on any MediaTek device.  It's just supposed to be audio firmware!

But OK.  It doesn't really hurt anything to change names.  Will respin again.

[1] And really can't be.  Not only am I not working on that stuff, it bears mentioning that I don't actually work for MediaTek and am only "authorized"[2] to be doing this in the context of "port largely-pre-existing SOF audio firmware to Zephyr" and not "Design a Zephyr integration for all MediaTek devices". 

[2] There's no formal authorization involved, the SOF code is mostly upstream (though I'm actually ahead of that process on 8196) and Apache-licensed.  But I absolutely don't want to stomp on any toes.